docs/conf.py

```diff
@@ -463,7 +463,7 @@
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3.5', None),
     'django': ('https://docs.djangoproject.com/en/1.10/', 'https://docs.djangoproject.com/en/1.10/_objects/'),
-    'model_utils': ('https://django-model-utils.readthedocs.io/en/latest/', None),
+    'model_utils': ('https://django-model-test_utils.readthedocs.io/en/latest/', None),
```

Comment:
Ugh. Yes it is... the IDE did it. My fault for not carefully reading the diff, though.